rosa_install_access_delete_clusters/rosa-sts-creating-a-cluster-with-customizations.adoc

```diff
@@ -8,7 +8,7 @@ toc::[]
 
 include::snippets/rosa-4-11-12-snippet.adoc[]
 
-Create a {product-title} (ROSA) cluster with the AWS Security Token Service (STS) using customizations. You can deploy your cluster by using {cluster-manager-first} or the ROSA CLI (`rosa`).
+Create a {product-title} (ROSA) cluster with the AWS Security Token Service (STS) using customizations. You can deploy your cluster by using {cluster-manager-first} or the {product-title} (ROSA) CLI (`rosa`).
```

Comment:
just ROSA CLI, product title is expanded earlier in the paragraph

Reply:
Maybe, "ROSA CLI (rosa)" here?